scripts/jquery.subsections.js

```diff
@@ -220,6 +220,36 @@ function addSubsection(event, ui) {
 	ui.item.value = "";
 }
 
+// Добавление раздела в хранимые, по нажатию на ид форума
+function addUnsavedSubsection(forum_id) {
+	$("#dialog").dialog(
+		{
+			buttons: [
+				{
+					text: "Да, добавить",
+					click: function () {
+						// Открываем вкладку настроек, настройки хранимых подразделов и вставляем ид раздела
+						$("#menutabs").tabs("option", "active", 1);
+						$("div.sub_settings").accordion("option", "active" , 3);
+						$("#add-forum").val(forum_id).autocomplete("search", forum_id);
+						$(this).dialog("close");
+					},
+				},
+				{
+					text: "Нет",
+					click: function() {
+						$(this).dialog("close");
+					}
+				}
+			],
+			modal: true,
+			resizable: false,
+			// position: [ 'center', 200 ]
```

Comment:
It's redundant here.

Reply:
![Screenshot_2022-12-20_09-29](https://user-images.githubusercontent.com/54838254/208598997-7b867bcd-81be-43b9-b7cb-a82891923517.png)
![Screenshot_2022-12-20_09-29_1](https://user-images.githubusercontent.com/54838254/208599003-af0ec676-a7fc-43f9-9332-19d46c0313d8.png)
![Screenshot_2022-12-20_09-30](https://user-images.githubusercontent.com/54838254/208599005-19764314-eb90-4438-96d2-28e6a328fabe.png)
![Screenshot_2022-12-20_09-30_1](https://user-images.githubusercontent.com/54838254/208599008-a88e1422-616f-4c67-95cc-631a0e27319d.png)

Reply:
I guess, @Orhideous means the commented `position` line.